docker-run.sh

```diff
@@ -54,13 +55,29 @@ EOF
 \$dolibarr_main_document_root_alt='/var/www/html/custom';
 \$dolibarr_main_data_root='/var/www/documents';
 \$dolibarr_main_db_host='${DOLI_DB_HOST}';
-\$dolibarr_main_db_port='${DOLI_DB_HOST_PORT}';
+\$dolibarr_main_db_port='3306';
 \$dolibarr_main_db_name='${DOLI_DB_NAME}';
 \$dolibarr_main_db_prefix='llx_';
 \$dolibarr_main_db_user='${DOLI_DB_USER}';
 \$dolibarr_main_db_pass='${DOLI_DB_PASSWORD}';
 \$dolibarr_main_db_type='mysqli';
+\$dolibarr_main_authentication='$auth';
 EOF
+    if [[ $auth =~ .*ldap.* ]]; then
```

Comment:
I would prefer a strict egality here ... approx could lead to misunderstanding or strange behaviour for end users.
```suggestion
    if [[ ${AUTH} =~ .*ldap.* ]]; then
```

Reply:
For this point, I mentionned earlier, the `dolibarr_main_authentication` can be equal (at least) to `dolibarr`, `ldap` or `ldap, dolibarr` / `dolibarr, ldap` to ensure a backup login for internal users in case of LDAP error.
That's why I did not implement a strict equality in this case.

You can see this documented here (just above the 'Usage' part, there is an example of configuration)
https://wiki.dolibarr.org/index.php/Module_LDAP#Usage